src/evaluators/ClassDeclaration.js

```diff
@@ -150,11 +150,15 @@ export function ClassDefinitionEvaluation(
     // 7. Let proto be ObjectCreate(protoParent).
     let proto = Create.ObjectCreate(realm, protoParent);
 
+    // Provide a hint that this prototype is that of a class
```

Comment:
Why only a hint?

Reply:
It is kind of a hint because it is a hint that we should try to serialize this as a class body instead of an object literal. It is possible that this doesn't need to be serialized as a class and that this is unnecessary (e.g. if it has no `super` calls). It is also possible that some functions that are not defined as classes in source would be better off being serialized as a class. E.g. a lowered Babel class can probably be turned back into a class.

We could determine either one based on some other heuristic but for now it is a hint.